tests/test_vrf.rb

```diff
@@ -107,6 +107,59 @@ def test_description
     v.destroy
   end
 
+  def test_mhost
+    v = Vrf.new('test_mhost')
+    t_intf = 'Loopback100'
+    if validate_property_excluded?('vrf', 'mhost_ipv4')
```

Comment:
This should be `mhost_default_interface` instead of `mhost_ipv4` after the yaml change right?